Processor_CashFlows_SftpCommunication.php

```diff
@@ -0,0 +1,64 @@
+<?php
+class Processor_CashFlows_SftpCommunication extends Jobs_SftpCommunication
```

Comment:
Please include relevant comments and documentation for this class and its members.

Reply:
Please include tests for this class and its members.